### ISSUE TITLE
feature testing: remove and kill all containers

### ISSUE THREAD

**Issue by charlierudolph** [Originate/exosphere#441]
Lets do this before starting tests since orphaned containers cause errors.

**Comment by charlierudolph** [comment]
@hugobho @mcharawi @cheahuychou thoughts?

**Comment by hugoesthere** [comment]
This only includes all containers listed in `docker-compose.yml` right? @mcharawi was working on adding a feature to `exo clean` that does essentially this. We'd just have to run `exo clean` against the main `docker-compose.yml` as well as the test `docker-compose.yml` file.

**Comment by charlierudolph** [comment]
closing in favor of #444

**Comment by charlierudolph** [comment]
This would need a disclaimer that running the exosphere feature tests will stop and remove all containers. This is because if you interrupt running feature tests you may be left with orphaned containers that can cause the exo-run to fail. @hugobho thoughts?

**Comment by hugoesthere** [comment]
Who/what is stopping and removing the containers?

**Comment by charlierudolph** [comment]
We would add some code to do that in a `BeforeSuite`

**Comment by hugoesthere** [comment]
Would we use the new `exo clean` command which reads your `docker-compose.yml` files and removes only those containers? If so then we'll only remove containers that exosphere touches and not the ones the user has running themselves. Sure we'll stop service containers if they have `exo run` running in the background but that's not as bad as stopping and removing their non-exosphere containers.

**Comment by charlierudolph** [comment]
This is only for running our feature tests when developing exopshere. I want to get our test suite reliably passing and then we can make this better about not removing all containers

**Comment by hugoesthere** [comment]
Ok sure, let's just put the disclaimer in for now then.

**Comment by charlierudolph** [comment]
Closing with won't fix. We can come back to this if needed
